packages/sdk/test/methods.spec.ts

```diff
@@ -182,6 +206,59 @@ describe('LookerNodeSDK', () => {
     await sdk.authSession.logout()
   }
 
+  describe('downloads', () => {
+    it('png and svg', async () => {
+      const sdk = new LookerSDK(session)
+      const looks = await sdk.ok(sdk.search_looks({ limit: 1 }))
+      let type = ''
+      let id = ''
+      expect(looks).toBeDefined()
+      if (looks.length > 0) {
+        type = 'look'
+        id = looks[0].id!.toString(10)
+      } else {
+        const dashboards = await sdk.ok(sdk.search_dashboards({ limit: 1 }))
+        expect(dashboards).toBeDefined()
+        if (dashboards.length > 0) {
+          type = 'dashboards'
+          id = dashboards[0].id!
+        }
+      }
+      expect(type).toBeDefined()
+      expect(id).toBeDefined()
```

Comment:
let me know if you have trouble getting the docker container running. I'm torn about this "defensive" testing :-) On the one hand assuming a look exists makes the test easier to read which I'd prefer. However, a previous test might have deleted all the looks? I guess in that case I'd rather we created a look than falling back to trying to find a dashboard all of which may have also been deleted?

IMHO we should just assume other test writers will not go around deleting pre-existing data that (unless it's critical and there's no way to create an object for deletion first). I'd say just assume a look exists and delete this fallback behavior.

Reply:
When we have a consistent, known setup we can update this test. For now, I want to ensure it's testing the pattern we want, which is both binary and string payloads.